[Edit] HTML Add Media: src (#3550)
Codecademy/docs: content/html/concepts/attributes/terms/src/src.md

| [`<img>`](https://www.codecademy.com/resources/docs/html/images)  | Embeds online photos, logos, and other pictures into an HTML file. |                                   
| [`<video>`](https://www.codecademy.com/resources/docs/html/elements/video)   | Embeds movie clips or other video sources into an HTML file.|
| [`<audio>`](https://www.codecademy.com/resources/docs/html/elements/audio)   | Embeds sound files on websites and applications. |
| [`<script>`](https://www.codecademy.com/resources/docs/html/elements/script) | Embeds JavaScript and other executive code or data into an HTML file. |
| [`<iframe>`](https://www.codecademy.com/resources/docs/html/iframes)   | Embeds an external web page into an HTML file. |
| [`<track>`](https://www.codecademy.com/resources/docs/html/elements/track ) | Specifies the subtitles and closed captions for `<audio>` and `<video>` elements. |

## Example 1: Image

The snippet below shows a logo being inserted into an HTML file:
The provided snippet demonstrates how the **`src`** attribute specifies the source of the image file, "logo.png," which will be displayed on the web page.

```html
<img src="logo.png" alt="Codecademy logo" />
```

> > **Note:** When using online images or images from the folders in an IDE workspace, always add `alt` text at the end of the `<img>` element just in case the browser has trouble finding them.

Reviewer: I would prefer you not delete this note.

> The provided snippet illustrates that the **`src`** attribute specifies the source of a video file, which will be used in conjunction with the [`<video>`](https://www.codecademy.com/resources/docs/html/elements/video) element for playback:

Reviewer: ```suggestion
The following snippet illustrates the `src` attribute specifying the source of a video file, which will be used in conjunction with the [`<video>`](https://www.codecademy.com/resources/docs/html/elements/video) element for playback:
```

></video>
```

> 

Reviewer: ```suggestion
```
Remove blank line

## Example 3: Audio

> The provided snippet illustrates that the **`src`** attribute  points to an audio file ("music.mp3") that can be played using the [`<audio>`](https://www.codecademy.com/resources/docs/html/elements/audio) element. The "controls" attribute adds audio controls like play, pause, and volume:

Reviewer: ```suggestion
The following snippet illustrates the `src` attribute  pointing to an audio file ("music.mp3") that can be played using the [`<audio>`](https://www.codecademy.com/resources/docs/html/elements/audio) element. The `controls` attribute adds audio controls like play, pause, and volume:
```

```html
<audio src="music.mp3" controls></audio>
```

## Example 4: Script

> The provided snippet illustrates that the **`src`** attribute is used in this case to link an external JavaScript file ("script.js") to the HTML document for executing client-side scripts:

Reviewer: ```suggestion
The following snippet illustrates the **`src`** attribute being used in this case to link an external JavaScript file ("script.js") to the HTML document for executing client-side scripts:
```
